doc/whats_new/v0.22.rst

```diff
@@ -50,6 +50,15 @@ Changelog
   value of the ``zero_division`` keyword argument. :pr:`15879`
   by :user:`Bibhash Chandra Mitra <Bibyutatsu>`.
 
+:mod:`sklearn.model_selection`
+..............................
+
+- |Fix| :class:`model_selection.GridSearchCV` and
+  :class:`model_selection.RandomizedSearchCV` accept scalar values provided in
```

Comment:
Do other things such as callable belong to "scalar values"?

Reply:
I think the change log entry understates the change, but that's okay, as this is relatively readable.